.gitattributes

```diff
@@ -38,68 +38,81 @@
 #/Changes                 merge=union
 
 # No header for text files (would be too obtrusive).
-*.md                     ocaml-typo=missing-header
-README*                  ocaml-typo=missing-header
-*.adoc                   ocaml-typo=missing-header,long-line,unused-prop
-
-/.mailmap                ocaml-typo=long-line,missing-header,non-ascii
-/.merlin                 ocaml-typo=missing-header
-/Changes                 ocaml-typo=utf8,missing-header
-/INSTALL                 ocaml-typo=missing-header
-/LICENSE                 ocaml-typo=long-line,very-long-line,missing-header
+*.md                     typo.missing-header
+README*                  typo.missing-header
+*.adoc                   typo.missing-header
+*.adoc                   typo.long-line=may
+
+/.mailmap                typo.long-line typo.missing-header typo.non-ascii
+/.merlin                 typo.missing-header
+/Changes                 typo.utf8 typo.missing-header
+/INSTALL                 typo.missing-header
+/LICENSE                 typo.long-line typo.very-long-line typo.missing-header
 # tools/ci/appveyor/appveyor_build.cmd only has missing-header because
 # dra27 too lazy to update check-typo to interpret Cmd-style comments!
-/tools/ci/appveyor/appveyor_build.cmd      ocaml-typo=long-line,very-long-line,missing-header,non-ascii text eol=crlf
-/tools/ci/appveyor/appveyor_build.sh       ocaml-typo=non-ascii
+/tools/ci/appveyor/appveyor_build.cmd       typo.long-line typo.very-long-line typo.missing-header typo.non-ascii
+/tools/ci/appveyor/appveyor_build.sh        typo.non-ascii
+/tools/ci/inria/remove-sinh-primitive.patch typo.white-at-eol typo.missing-header
 
+# ignore auto-generated .depend files
+.depend                  typo.prune
 
-asmcomp/*/emit.mlp       ocaml-typo=tab,long-line,unused-prop
-asmcomp/power/NOTES.md   ocaml-typo=missing-header
+# Makefiles may contain tabs
+Makefile*                typo.tab=may
 
-/config/gnu              ocaml-typo=prune
+asmcomp/*/emit.mlp       typo.tab=may typo.long-line=may
+asmcomp/power/NOTES.md   typo.missing-header
 
-emacs/*.el               ocaml-typo=long-line,unused-prop
-emacs/caml.el            ocaml-typo=long-line,unused-prop,missing-header
-emacs/COPYING            ocaml-typo=tab,non-printing,missing-header
-emacs/ocamltags.in       ocaml-typo=non-printing
+/config/gnu              typo.prune
 
-/experimental            ocaml-typo=prune
+emacs/*.el               typo.long-line=may
+emacs/caml.el            typo.long-line=may typo.missing-header
+emacs/COPYING            typo.tab typo.non-printing typo.missing-header
```

Comment:
This one should be `typo.prune`: we're not ever going to change the GPL text, even if we invent new typo checks or whatever.